pepper-apis/src/test/java/org/broadinstitute/ddp/service/PdfGenerationServiceTest.java

```diff
@@ -69,12 +70,15 @@
 
     @BeforeClass
     public static void setup() throws Exception {
+        System.out.println("About to sleep during setup");
+        Thread.sleep(30000);
```

Comment:
Are we sure we want to sleep during the test suite? And 30 seconds at that...

Reply:
Those should not be there. Removed them (was holding up test so I could attach Profiler my test code executed).
b24e1e52